subjects/cybersecurity/image-inspector/audit/README.md

```diff
@@ -0,0 +1,96 @@
+- Source code for the Image Inspector tool.
+- Any required configuration files and scripts for running the tool.
+
+###### Are all the required files present?
+
+##### Play the Role of a Stakeholder
+
+Organize a simulated scenario where the student takes on the role of a Digital Forensics Expert and explains their solution and knowledge to a team or stakeholder. Evaluate their grasp of the concepts and technologies used in the project, their communication efficacy, and their critical thinking about their solution and the knowledge behind this project.
+
+Suggested role play questions include:
+
+- What is metadata in the context of digital images, and why is it important?
+- How does steganography work, and what are its potential uses and risks?
+- What challenges did you face while developing the Image Inspector tool, and how did you address them?
+- How does the Image Inspector tool help in identifying hidden information within images?
+- How can this tool be used in real-life digital forensics or cybersecurity scenarios?
+- What ethical considerations should be taken into account when analyzing images for hidden data?
+
+###### Were the students able to answer all the questions?
+
+###### Did the students demonstrate a thorough understanding of the concepts and technologies used in the project?
+
+###### Were the students able to communicate effectively and justify their decisions and explain the knowledge behind this project?
+
+###### Were the students able to evaluate the value of this project in real-life scenarios?
+
+###### Did the students demonstrate an understanding of ethical and legal considerations related to digital forensics?
+
+##### Check the Student Documentation in the `README.md` File
+
+###### Does the `README.md` file contain all the necessary information about the tool (prerequisites, setup, configuration, usage, ...)?
+
+###### Does the `README.md` file contain clear guidelines and warnings about the ethical and legal use of the tool?
+
+##### Review the Tool's Design and Implementation
+
+1. **Help Command:**
+
+```sh
+$> image-inspector --help
+```
+
+###### Does the output include an explanation of how to use the tool, with all options clearly described?
+
+2. **Metadata Extraction Option:**
+
+```sh
+$> image-inspector -m image-example.jpeg -o metadata.txt
+```
+
+###### Does the output correctly extract and display metadata such as geolocation, device information, and date/time?
+
+###### Is the output stored in the file specified in the output parameter?
+
+3. **Steganography Detection Option:**
+
+```sh
+$> image-inspector -s image-example.jpeg -o hidden_data.txt
+```
+
+###### Does the output correctly detect and extract any hidden PGP keys or other concealed information within the image?
+
+###### Is the output stored in the file specified in the output parameter?
+
+##### Testing with Images
+
+**You will be provided with an example image to test the students tool. Feel free to test with other images.**
```

Comment:
Its better to provide more than one image with its correct output to make sure that the tool did the extraction correctly, and to avoid mis testing by the auditor.

Reply:
done, 3 images with different PGP keys added

Reply:
Can you please provide also the correct output of each of the following images to ease the correction process for the auditor

Reply:
Done in this PR: https://github.com/01-edu/public/pull/2731